repository/t.json

```diff
@@ -996,6 +996,16 @@
 				}
 			]
 		},
+		{
+			"name": "TextTree",
+			"details": "https://github.com/gbif/text-tree-sublime",
+			"releases": [
```

Comment:
Even though this is not a "language" in the traditional sense it would still make sense to add the "[language syntax](https://packagecontrol.io/browse/labels/language%20syntax)" label.

Reply:
Would it be ok to ship with the color scheme, but remove the default use of it in `TextTree.sublime-settings`?

Reply:
Sure, that’s totally fine.

Reply:
Remove in https://github.com/gbif/text-tree-sublime/releases/tag/1.0.3

Reply:
Don’t forget to add the label hete though!

Reply:
added